src/components/existing-request/existing-request-display.vue

```diff
@@ -427,6 +432,25 @@ export default class ExistingRequestDisplay extends Mixins(
             this.isConsentUnRequired)
   }
 
+  get showAlterNowButton (): boolean {
+    return this.nr.request_action_cd === NrRequestActionCodes.CONVERSION &&
+          this.nr.state === NrState.APPROVED
+  }
+
+  get isAllowAlterOnline (): boolean {
+    return !(this.nr.requestTypeCd === NrRequestTypeCodes.CONVERT_BEN ||
+      this.nr.requestTypeCd === NrRequestTypeCodes.CONVERT_CORP ||
+      this.nr.requestTypeCd === NrRequestTypeCodes.CONVERT_ULBE)
+  }
+
+  get isOpenExternal (): boolean {
+    if (this.showAlterNowButton) {
+      const supportedAlterationTypes = GetFeatureFlag('supported-alteration-types')
+      return !supportedAlterationTypes.includes(this.nr.requestTypeCd)
+    }
+    return false
+  }
```

Comment:
Please rename this to something that tells me what this value is. Maybe "isSupportedAlteration"?

See also ComminMixin::isSupportedEntity.

Should ine 449 check `requestTypeCd` or `entity_type_cd`?

Reply:
It should check `requestTypeCd`.

Reply:
I like that you've moved this logic to CommonMixin.

I'd still like to see "isOpenExternal" named something more specific.